lib/resources/secure_storage.dart

```diff
@@ -26,6 +27,7 @@ class SecureStorageKeys {
   static const publicIdsList = "${prepend}_PIDs"; // The public IDs
   static const namesList = "${prepend}_NAMEs"; // The names of the IDs
   static const settings = "${prepend}_SETTINGS"; // The settings of the wallet
+  static const pendingTransactions = "${prepend}_PENDING";
 }
```

Comment:
please call it "${prepend}_PENDING_TRX"

Reply:
It should be removed because we don't currently depend on secure storage for storing pending transactions.

Resolved.